openassessment/xblock/load_static.py

```diff
@@ -0,0 +1,44 @@
+"""
+Helper class for loading generated file from webpack.
+"""
+import json
+import logging
+
+from pkg_resources import resource_string
+from django.conf import settings
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class LoadStatic:
+    """
+    Helper class for loading generated file from webpack.
+    """
+
+    _manifest = dict()
+    _base_url = ''
+
+    @staticmethod
+    def reload_manifest():
+        """
+        Reload from manifest file
+        """
+        json_data = resource_string(__name__, 'static/dist/manifest.json').decode("utf8")
+        LoadStatic._manifest = json.loads(json_data)
+        if LoadStatic._manifest['is_dev_server']:
+            host = ''
+        elif hasattr(settings, 'LMS_ROOT_URL'):
+            host = settings.LMS_ROOT_URL
```

Comment:
Assuming this variable is being use correctly. This is one of the reason the older code was using `add_javascript` instead of `add_javascript_url`. CMS could not get the script because there isn't iframe for proxy to CND.